tests/integration_tests/scheduler/bin/bhist.py

```diff
@@ -48,6 +49,31 @@ def bhist_formatter(jobstats: List[Job]) -> str:
     return string
 
 
+def bhist_long_formatter(jobstats: List[Job]) -> str:
+    formatted_job_outputs = []
+    for job in jobstats:
+        job_output = dedent(
+            f"""
+            Job <{str(job.job_id)}>, Job Name <{str(job.job_name)}>, User <{str(job.user)}>,
+                                Project <default>, Command <sleep 100>
+            Mon Apr 19 11:32:57: Submitted from host <test_host>, to Queue <normal>, CWD <$
```

Comment:
Mention in docstring that the data from this function is arbitrary? (up until now, the the mocked bhist.py gives correct data, read from the filesystem)